### PR TITLE
Essential requirements evidence

### PR DESCRIPTION
**Pull request by idavidmcdonald**
[Pivotal story](https://www.pivotaltracker.com/story/show/129841565)

- Pulls in latest frameworks repo giving us a new page for the new `essentialRequirements` dynamic list question
- Corresponding tests for the new page
- Uses the content loader `unformat_data` to transform brief response data into form data (see nice example of unformatting [here](https://github.com/alphagov/digitalmarketplace-content-loader/blob/master/dmcontent/questions.py#L315) in the content loader)

screen shot 2017-01-04 at 12 26 14

### CODE REVIEW

File: tests/app/main/test_briefs.py [hunk 1]
```diff
@@ -676,11 +676,9 @@ def test_essential_requirements_evidence_escapes_brief_data(self):
         doc = html.fromstring(res.get_data(as_text=True))
         questions = doc.xpath(
             "//*[@class='question-heading']")
-        # if item is excaped correctly it will appear as text rather than an element
+
         assert questions[0].find("h1") is None
-        assert questions[0].text.strip() == '&lt;h1&gt;Essential one with xss&lt;/h1&gt;'
```
Comment (galund) [inline tests/app/main/test_briefs.py]: Hang on, if they are appearing as literal '<h1>' then they *aren't* being escaped, this sounds like the test is finding a bug?

Comment (Wynndow) [inline tests/app/main/test_briefs.py]: Is my understanding of 'escaped' off? Do html tags need to be converted to special characters?

Comment (idavidmcdonald) [inline tests/app/main/test_briefs.py]: But does the assertion above confirm there is no h1 element (which would imply xss)?

Comment (idavidmcdonald) [inline tests/app/main/test_briefs.py]: image
Have checked manually and they are not rendered as html elements but rendered as a string which means we have not been xss'd?

**Comment by tombye** [comment]
👍

**Pull request merged**